src/Chamilo/CoreBundle/Entity/Course.php

```diff
@@ -282,12 +317,128 @@ class Course
      */
     protected $room;
 
+    /**
```

Comment:
Database changes are not allowed in 1.11.x

Reply:
Existing databases do not need to be updated, it will work as is.

Reply:
Databases do not need to be altered to get this to work.
New databases will get new referential integrity checks (foreign keys). No table structural change.